modules/serverless-ossm-enable-sidecar-injection-with-kourier.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// * serverless/service_mesh/serverless-ossm-with-kourier.adoc
+
+[id="serverless-ossm-enable-sidecar-injection-with-kourier_{context}"]
+= Enabling sidecar injection for a Knative service
+
+You can add the `sidecar.istio.io/inject="true"` annotation to a `Service` resource to enable sidecar injection for a Knative service.
+
+[IMPORTANT]
+====
+Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
```

Comment:
```suggestion
Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported with Kourier enabled.
Please refer to another chapter "Integrating Service Mesh and OpenShift Serverless without Kourier" for that.
```
I wanted to say it is now supported by the instruction in https://deploy-preview-33179--osdocs.netlify.app/openshift-enterprise/latest/serverless/service_mesh/serverless-ossm-without-kourier.html. Please fix my bad expression :sweat:

Reply:
No problem @nak3 😄 thank you so much for the correction, I'll update it 🎉